plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/config/CubridMessages.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.ui.config;
+
+public class CubridMessages {
+    public static String dialog_struct_create_procedure_container = "Owner";
+    public static String dialog_struct_create_procedure_label_name = "Name";
+    public static String dialog_struct_create_procedure_combo_type = "Type";

```

Comment:
Don't do that. Use the `NLS` class for providing localization. Look up its usages in other parts of the application.

Reply:
Waiting for this to be fixed

Reply:
I have just fixed this. Thank you.